app/src/main/java/com/github/siela1915/bootcamp/ReplyViewHolder.java

```diff
@@ -1,29 +1,98 @@
 package com.github.siela1915.bootcamp;
 
+import static androidx.appcompat.content.res.AppCompatResources.getDrawable;
+
 import android.view.View;
 import android.widget.Button;
+import android.widget.CompoundButton;
 import android.widget.EditText;
 import android.widget.ImageView;
 import android.widget.TextView;
+import android.widget.Toast;
+import android.widget.ToggleButton;
 
 import androidx.annotation.NonNull;
 import androidx.recyclerview.widget.RecyclerView;
 
 import com.github.siela1915.bootcamp.Recipes.Comment;
+import com.github.siela1915.bootcamp.Recipes.Recipe;
+import com.github.siela1915.bootcamp.firebase.Database;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.database.FirebaseDatabase;
 
-public class ReplyViewHolder extends RecyclerView.ViewHolder {
+public class ReplyViewHolder extends RecyclerView.ViewHolder implements CompoundButton.OnCheckedChangeListener{
 
     TextView reply, likes, userName;
     ImageView photo;
-    //Comment parent;
```

Comment:
it is better to have private class attributs and to have getters and setters if needed